contribs/gmf/src/options.js

```diff
@@ -157,4 +162,97 @@
  * If true, you should use the externalMessage and the externalVisibility too.
  */
 
+/**
+ * Configuration option for {@link import("gmf/query/gridComponent.js").default} to merge
+ * grid tabs.
+ *
+ * E.g. `'two_wheels_park': ['velo_park', 'moto_park']}` merges the sources
+ * with label `velo_park` and `moto_park` into a new source `two_wheels_park`.
+ *
+ * @typedef {Object<string, string[]>} GridMergeTabs
+ */
+
+/**
+ * @typedef {Object} Fill
+ * @property {number[]} [color] The color.
+ */
+
+/**
+ * @typedef {Object} Stroke
+ * @property {number[]} [color] The color.
+ * @property {CanvasLineCap} [lineCap='round'] Line cap style: `butt`, `round`, or `square`.
+ * @property {CanvasLineJoin} [lineJoin='round'] Line join style: `bevel`, `round`, or `miter`.
+ * @property {number[]} [lineDash] Line dash pattern. Default is `null` (no dash).
+ * Please note that Internet Explorer 10 and lower do not support the `setLineDash` method on
+ * the `CanvasRenderingContext2D` and therefore this option will have no visual effect in these browsers.
+ * @property {number} [lineDashOffset=0] Line dash offset.
+ * @property {number} [miterLimit=10] Miter limit.
+ * @property {number} [width] Width.
+ */
+
+/**
+ * @typedef {Object} Circle
+ * @property {Fill} [fill] Fill style.
+ * @property {number} radius Circle radius.
+ * @property {Stroke} [stroke] Stroke style.
+ * @property {number[]} [displacement=[0,0]] displacement
+ */
+
+/**
+ * The style description.
+ * @typedef {object} Style
+ * @property {Fill} [fill] The fill color.
+ * @property {Stroke} [stroke] The stoke config.
+ * @property {Circle} [circle] The circle config.
+ * @property {number} [zIndex] The z index.
+ */
+
+/**
+ * @param {olStyle|Style} styleDescriptor The description of the style
+ * @returns {olStyle}
+ */
+export function buildStyle(styleDescriptor) {
```

Comment:
@fredj Is there nothing to do that in ol ? Do you know ?

Reply:
non, nothing like that in OpenLayers

Reply:
Thanks (could be an useful tool)